Mage.Tests/src/test/java/org/mage/test/cards/single/dsk/UnableToScreamTest.java

```diff
@@ -0,0 +1,125 @@
+package org.mage.test.cards.single.dsk;
+
+import mage.abilities.mana.GreenManaAbility;
+import mage.constants.*;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * @author markort147
+ */
+public class UnableToScreamTest extends CardTestPlayerBase {
+
+    public static final String UNABLE_TO_SCREAM = "Unable to Scream";
```

Comment:
It's better to write related card's ability text in the test too. I prefer to write it before `addCard` for each test, but it can be used before card name constant too. It helps with test readability without IDE's tools or on the github website.